docs/INTRODUCTION.md

```diff
@@ -1,63 +1,57 @@
-
 # The Open Visualisation Environment (OVE)
 
 OVE was developed to meet the requirements of controlling the [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) at the [Data Science Institute](https://www.imperial.ac.uk/data-science/) of [Imperial College London](https://www.imperial.ac.uk), but it is not specialized for that purpose.
-It can be used for visual analytics on Large High Resolution Displays, for presentations, or for collaborative groupwork.
-
-It allows a user to control the display of content in web browsers distributed across multiple computers.
-It does this by implementing a microservices architecture that allows the distributed execution of applications using web technologies.
-
 
-The main components of OVE are  **OVE Core**, which controls sections and the applications running within them and **OVE Core Apps**, which provide a set of useful applications for common tasks such as displaying images, displaying webpages, and drawing graphs.
-**OVE Services** provides core functionality microservices within OVE such as geometry.  **OVE Asset Services** provides services to divide large networks or images into tiles, and serve these to the corresponding applications.
+OVE can be used for visual analytics on Large High Resolution Displays, for presentations, or for collaborative group work. It allows a user to control the display of content in web browsers distributed across multiple computers by implementing a microservices architecture that allows the distributed execution of applications using web technologies.
 
+The main components of OVE are [**OVE Core**](https://github.com/ove/ove), which controls sections and the applications running within them and [**OVE Apps**](https://github.com/ove/ove-apps), which provide a set of useful applications for common tasks such as displaying images, displaying webpages, and drawing graphs.
 
-## Concepts
+[**OVE Services**](https://github.com/ove/ove-services) provides core functionality microservices within OVE such as the layout service. [**OVE Asset Services**](https://github.com/ove/ove-asset-services) provides an Asset Manager along with a collection of Asset Processing Services, used to upload archives or divide large networks or images into tiles, and serve these to the corresponding applications.
 
-OVE supports multiple ``spaces``. Each ``space`` is associated with one or more monitors, which may be attached to different computers, that together form a single display. This display may be a Large High Resolution Display (the [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution and consists of 64 monitors), but OVE is also suitable for use on much smaller displays.
+OVE also provides [graphical editors](https://github.com/ove/ove-editor) and [software development kits](https://github.com/ove/ove-sdks) that can be used to design and develop visualisation projects.
```

Comment:
"visualisation projects" - projects aren't necessarily visualizations, which is why we decided to call them *projects* rather than *visualizations*